src/Microsoft.AspNet.OData.Shared/Common/SRResources.resx

```diff
@@ -121,7 +121,7 @@
     <value>Cannot serialize a null '{0}'.</value>
   </data>
   <data name="TypeCannotBeSerialized" xml:space="preserve">
-    <value>'{0}' cannot be serialized using the {1}.</value>
+    <value>'{0}' cannot be serialized using the ODataMediaTypeFormatter.</value>
```

Comment:
Most of the places that use this string pass typeof(ODataMediaTypeFormatter) and most of those are in Shared code. ODataMediaTypeFormatter will be an AspNet-only class since it deals with AspNets-specific classes like MediaTypeFormatter and MediaTypeHeaderValue.